vmc.sh

```diff
@@ -8,21 +8,38 @@ build_requires:
   - CMake
   - "Xcode:(osx.*)"
   - alibuild-recipe-tools
+prepend_path:
+  ROOT_INCLUDE_PATH: "$VMC_ROOT/include/vmc"
 ---
 #!/bin/bash -e
-cmake "$SOURCEDIR"                             \
-  -DCMAKE_CMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE} \
-  -DCMAKE_INSTALL_PREFIX="$INSTALLROOT"        \
-  -DCMAKE_INSTALL_LIBDIR=lib                   \
+
+root_version=$(root-config --version)
+root_version=${root_version:0:1}
```

Comment:
There is already ROOT_VERSION which is defined by alibuild the moment you require root.

Reply:
I used that as suggested